doc/source/train/api/api.rst

```diff
@@ -128,7 +46,23 @@ PyTorch Training Loop Utilities
 .. _train-lightning-integration:
 
 PyTorch Lightning
-~~~~~~~~~~~~~~~~~
+*****************
+
+.. autosummary::
+    :toctree: doc/
+
+    ~train.lightning.prepare_trainer
+    ~train.lightning.RayLightningEnvironment
+    ~train.lightning.RayDDPStrategy
+    ~train.lightning.RayFSDPStrategy
+    ~train.lightning.RayDeepSpeedStrategy
+    ~train.lightning.RayTrainReportCallback
+
+.. note::
+
+    We will deprecate `LightningTrainer`, `LightningConfigBuilder`,
+    `LightningCheckpoint`, and `LightningPredictor` in Ray 2.8. Please 
+    refer to the :ref:`migration guides <migration-guide>` for more info.
```

Comment:
Some of the files are duplicated below.

Reply:
OK let me remove the transformers and accelerate ones .

Reply:
Oh I mean lines 67-85 are mostly duplicated as above.
<img width="406" alt="image" src="https://github.com/ray-project/ray/assets/3967392/14fc0c54-1848-4b2a-8676-b06fd9fe3f6a">

Reply:
Oh I see. Nice catch! I've removed the duplicated ones.